.vsts-dotnet-ci.yml

```diff
@@ -26,11 +29,37 @@ phases:
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Linux
+    name: Centos
     buildScript: ./build.sh
+    customMatrixes:
+      Build_Debug_Intrinsics:
+        _configuration: Debug-Intrinsics
+        _config_short: DI
+        _includeBenchmarkData: false
+      Build_Release:
+        _configuration: Release
+        _config_short: R
+        _includeBenchmarkData: true
+    queue:
+      name: Hosted Ubuntu 1604
+      container: CentosContainer
+
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: Ubuntu
+    buildScript: ./build.sh
+    customMatrixes:
+      Build_Debug_Intrinsics:
+        _configuration: Debug-Intrinsics
```

Comment:
I thought we were going to switch to the `Release` to use `3.0`/`Intrinsics`. That way we get Debug and Release coverage across both versions:

```
CentOS|Debug|3.0
CentOS|Release|2.1
Ubuntu|Debug|2.1
Ubuntu|Release|3.0
```

Reply:
sorry I didn't saw it correctly, I will correct it

Reply:
Moving back the CI to netcoreap2.1 untill the cli people resolves the https://github.com/dotnet/cli/issues/10802